datamodel/app/functions/audit_functions.sql

```diff
@@ -0,0 +1,295 @@
+-----------------------------------------------
+-----------------------------------------------
+-- Provide audit functions for pg_history_viewer
+-----------------------------------------------
+-----------------------------------------------
+
+CREATE OR REPLACE FUNCTION tww_sys.if_modified_func() RETURNS TRIGGER AS $body$
```

Comment:
you should not create stuff in tww_sys from app!

Reply:
Better move audit function into changelogs